src/components/Section/Foreign/Activities/RealEstateActivity/RealEstateActivity.test.jsx

```diff
@@ -136,15 +136,15 @@ describe('The RealEstateActivity component', () => {
     })
   })
 
-  it('Renders with no', () => {
+  xit('Renders with no', () => {
     const expected = {
       HasInterests: { value: 'No' }
     }
     const component = createComponent(expected)
     expect(component.find('.accordion').length).toBe(0)
   })
 
-  it('Renders full validated', () => {
+  xit('Renders full validated', () => {
```

Comment:
I just started fixing these tests and I remember why I commented them out now. There's something strange going on in these tests with the imports/exports. It keeps complaining about random components that are `undefined`. It's been complaining about `Branch, `Location`, BranchCollection`, etc.

It's strange because these components are shared  and imported throughout the application and I'm not sure why it's failing on these specific tests.

Reply:
Ah ok. I looked into this a little bit and I think I experienced a similar issue, which is I believe due to some circular dependencies in the tangled mess of Form components.. hopefully something we can resolve later on :(